framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -2374,6 +2382,17 @@ public final boolean isRelevant(AnnotatedTypeMirror tm) {
     return isRelevant(tm.getUnderlyingType());
   }
 
+  /**
+   * Returns true if qualifiers may be written on this. If this returns false, the qualifiers are
+   * not checked.
+   *
+   * @param tm a type
+   * @return true if users can write type annotations from this type system on the given type
+   */
+  public boolean canBeAnnotated(AnnotatedTypeMirror tm) {
```

Comment:
It's not clear how this is different from `isRelevant`.

Reply:
It isn't any more.  I actually have a branch that removes it; I'm waiting for CI to run on that branch.